doc-content/enterprise-only/getting-started/filtering-process-instances-proc.adoc

```diff
@@ -0,0 +1,49 @@
+[id='filtering-process-instances-proc']
+= Filtering process instances using default or advanced filters
```

Comment:
This topic has already been reviewed as part of:
https://github.com/gadeyredhat/kie-docs/pull/12